manoseimas/mps_v2/json_urls.py

```diff
@@ -14,4 +14,6 @@
     url(r'^discussion/(?P<statement_id>\d+)/?$',
         views.mp_discussion_json, name='mp_discussion_json'),
     url(r'^suggesters/?$', views.suggesters_json, name='suggesters_json'),
+    url(r'^resolutions/(?P<suggester_slug>.+)/?$',
```

Comment:
I want this fixed before the release, or at least tested that it works correcly with both

    http://localhost:8000/.../statements/some-slug

and

    http://localhost:8000/.../statements/some-slug/